config-linux.md

```diff
@@ -158,13 +158,20 @@ cgroups provide controls to restrict cpu, memory, IO, pids and network for the c
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+If `cgroupsPath` is
+* ... an absolute path (starting with `/`), the implementation MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
```

Comment:
On Thu, Jun 09, 2016 at 05:42:57PM -0700, Aleksa Sarai wrote:
> > +* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
> 
> I prefer SHOULD, since I believe a runtime should have a good reason
> for it to not do that way…

Is which leaf of a random runtime-generated directory you're at all
that important to you?  I don't see a lot of space between MAY and
SHOULD here.